Utilities/GetPSGalleryItemsForAuthor.ps1

```diff
@@ -0,0 +1,100 @@
+﻿
+<#PSScriptInfo
+
+.VERSION 0.1.0
+
+.GUID 788b94eb-f6f0-400d-b259-2e15c3f102ee
+
+.AUTHOR J. Keith Bankston
+
+.COMPANYNAME Microsoft
+
+.COPYRIGHT Microsoft, 2018
+
+.TAGS GalleryUtility,GDPR
+
+.LICENSEURI 
+
+.PROJECTURI 
+
+.ICONURI 
+
+.RELEASENOTES
+  Version 0.1.0 - Initial Version. See description for use information. 
+
+
+.PRIVATEDATA 
+
+#>
+
+<# 
+
+.DESCRIPTION 
+This script will save all versions of all scripts and modules published to the PowerShell Gallery by a given Author.
+The Author is specified in the manifest for each item, and is expected to be consistent for a given user.
+
+The script assumes that there is at least 1 version of every item that is listed. 
+This can be verified by logging into the PowerShell Gallery, navigating to https://www.powershellgallery.com/account/Packages.
+If there are items that are hidden, they will be shown at the bottom of that page under the header: "Unlisted Items"
+
+Use instructions: 
+* Provide the AuthorName, which is the Author used in the item manifest. 
+  Note that the Author is not guaranteed to match the Owner, which is the account name for those who publish the item.
+* Provide the LocalFolder to place all the PowerShell Gallery items into. 
+
+For questions on this script, email cgadmin@microsoft.com 
+
+#> 
```

Comment:
Please consider including a sample/example here.

Reply:
Done in pending PR. Thanks.